test_api.py

```diff
@@ -159,7 +161,11 @@ def test_scenario_cancel_running_finished_ok():
     # wait until the job has stopped
     listjobs_wait(jobid, 'finished')
     jobinfo = assert_listjobs(finished=jobid)
-    assert jobinfo == { 'id': jobid, 'project': RUN_PROJECT, 'spider': RUN_SPIDER, 'state': 'finished' }
+    assert jobinfo['id'] == jobid
+    assert jobinfo['project'] == RUN_PROJECT
+    assert jobinfo['spider'] == RUN_SPIDER
+    assert jobinfo['state'] == 'finished'
+    assert datetime.strptime(jobinfo['start_time'], TIME_FORMAT)
```

Comment:
I'd rather keep the previous form, so that if anything is added to the dict, you are aware that a test needs changing.

Reply:
I will update this. Would need to look a bit more into mocking the `start_time` and `end_time` of the jobs during tests, since a dict comparison will require the exact `start_time` and `end_time` of the container/pods. Or maybe a different solution.

Reply:
Ah, that makes sense.
What about removing the time fields from the dict under test, and testing these separately? That would both allow us to test the full response, and allow proper time comparison.
Or use a time-freezing testing library (there must exist one).

Reply:
I tried using `freezegun` to set a specific time during testing, but since we are talking to the kubernetes/docker APIs and not using the datetime module, this solution did not work. I will do a bit more digging. The first solution should also work though, but it might be better to see if there are alternative solutions.

Reply:
Ah, that totally makes sense! I'm not for mocking the result in the component under test, so let's fix this in the testing part.